src/Deputy/Subscriber/BindAdherentDistrictSubscriber.php

```diff
@@ -28,17 +28,19 @@ public function updateReferentTagWithDistrict(AdherentEvent $event): void
         $adherent = $event->getAdherent();
 
         if ($adherent->isGeocoded()) {
-            $referentTags = $this->districtRepository->findDistrictReferentTagByCoordinates(
+            $districts = $this->districtRepository->findDistrictsByCoordinates(
                 $adherent->getLatitude(),
                 $adherent->getLongitude()
             );
 
-            if (!empty($referentTags)) {
-                foreach ($referentTags as $referentTag) {
-                    if ($adherent->getReferentTags()->contains($referentTag)) {
+            if (!empty($districts)) {
+                foreach ($districts as $district) {
+                    if (!\in_array($adherent->getCountry(), $district->getCountries(), false)) {
```

Comment:
`false` it's a default value of in_array

Reply:
resolved